docs/code-scanning-tools/sonar.md

```diff
@@ -1,22 +1,26 @@
 ---
-title: "Sonar"
+title: "SonarCloud"
 sidebar_position: 1
 ---
 
-# Sonar
+# SonarCloud
 
-Pixeebot can automatically fix issues detected by [Sonar](https://www.sonarsource.com/products/sonarcloud/). This page explains how to integrate your Sonar results with Pixeebot.
+Pixeebot can automatically fix issues detected by [SonarCloud](https://www.sonarsource.com/products/sonarcloud/). This page explains how to integrate your SonarCloud results with Pixeebot.
+
+:::info
+Looking to fix [SonarQube](https://www.sonarsource.com/products/sonarqube/) issues? See [SonarQube](./sonarqube.md)
+:::
 
 ## How to Get Started
 
-1. Install [pixeebot Github Application](https://github.com/marketplace/pixeebot-automated-code-fixes)
-2. Install [Sonar Github Application](https://github.com/marketplace/sonarcloud)
-3. Add [Upload Tool Results Github Action](https://github.com/marketplace/actions/upload-tool-results-to-pixeebot)
-4. Start receiving Pull Requests that address Sonar findings.
+1. Install [Pixeebot GitHub Application](https://github.com/marketplace/pixeebot-automated-code-fixes)
+2. Install [Sonar GitHub Application](https://github.com/marketplace/sonarcloud)
```

Comment:
Nitpick but we can just use `1.` here and below. I know this wasn't your change originally.

Reply:
I too prefer to use 1. for each bullet, but something in my / our tool chain is unhappy with it. I forget the details, but I didn't try to fight it.